taxonomies/categories.txt

```diff
@@ -48279,6 +48387,9 @@ fr:Extrait de vanille
 nl:Vanille-extracten
 pt:Extracto de vainilla
 wikidata:en:Q10749869
+agribalyse_proxy_food_code:en:11065
+agribalyse_proxy_food_name:en:Vanilla, alcoholic extract
```

Comment:
exact match instead of proxy?

Reply:
there are 2 sub-categories according to processing below